src/classes/BDI_Donations_TEST.cls

```diff
@@ -530,6 +530,191 @@ public with sharing class BDI_Donations_TEST {
         system.assertEquals(label.bdiMatched, listDI[1].PaymentImportStatus__c);
     }
 
+    /*********************************************************************************************************
+    * @description operation
+    *        import existing contacts with existing open Opportunities
+    * verify:
+    *       opportunity objects matched & closed
+    *       payment objects updated
+    *       user-selected payment status preserved
+    */
+    static testMethod void TwoDIWithUserSelectedExistingDonations() {
```

Comment:
I assume this was copied from another test method, but we should at least fix the styles on newly created tests.